<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NoiseInfo</class>
 <widget class="QDialog" name="Dialog_NoiseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Показатели</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recources.qrc">
    <normaloff>:/recources/actions/actionInfo.png</normaloff>:/recources/actions/actionInfo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_nc">
     <property name="toolTip">
      <string>Нормированная взаимная корреляция</string>
     </property>
     <property name="text">
      <string>NC = </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_cq">
     <property name="toolTip">
      <string>Качество корреляции</string>
     </property>
     <property name="text">
      <string>CQ = </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_mse">
     <property name="toolTip">
      <string>Среднеквадратическая ошибка</string>
     </property>
     <property name="text">
      <string>MSE = </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_snr">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Соотношение сигнал-шум</string>
     </property>
     <property name="text">
      <string>SNR = </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_NoiseInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_NoiseInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
